stack.cabal

```diff
@@ -219,7 +219,7 @@ library
                    , vector-binary-instances
                    , yaml >= 0.8.10.1
                    , zlib >= 0.5.4.2 && < 0.7
-                   , deepseq >= 1.4
+                   , deepseq >= 1.3 && < 1.5
```

Comment:
I'm not quite sure how `stack-7.8.yaml` does manage to circumvent the dependency trouble regarding `deepseq-1.3`, but somehow it does, so I might as well remove this change.